athf/mcp/tools/investigate_tools.py

```
@@ -0,0 +1,79 @@
+                fm = parsed.get("frontmatter", {})
+                if investigation_type and fm.get("type", "").lower() != investigation_type.lower():
+                    continue
+                results.append({
+                    "investigation_id": fm.get("investigation_id", f.stem),
+                    "title": fm.get("title", ""),
+                    "type": fm.get("type", ""),
+                    "status": fm.get("status", ""),
+                    "tags": fm.get("tags", []),
+                })
+            except Exception:
+                continue
```

Ruff (0.15.6)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)